### DISCUSSION TITLE
Failed to call any service

### DISCUSSION THREAD

**Discussion by Disincentivato** [alandtse/tesla#976] [Q&A]
Hi,
from yesterday evening my integration seems stop working.
I can get the vehicle information but it's impossible to send any command at the car. Any suggestion? 
![Immagine 2024-05-28 121843](https://github.com/alandtse/tesla/assets/90531670/44d154e7-46dd-4a09-90de-34b7cb316c14)

**Comment by pmrosa** [comment]
Me too since Sunday or so.

**Comment by bbrendon** [comment]
I tesla is slowly forcing everyone to the fleet api.
https://community.home-assistant.io/t/tesla-custom-integration-fleet-api/705029

**Reply by Boogmeister** [reply]
I'm on Fleet API and it was working until tonight.

Now as above can pull data, cannot send commands.

I have always been on the Fleet API.

Is there some sort of troubleshooting steps to try to figure out where the problem is?

**Reply by Boogmeister** [reply]
> I'm on Fleet API and it was working until tonight.
> 
> Now as above can pull data, cannot send commands.
> 
> I have always been on the Fleet API.
> 
> Is there some sort of troubleshooting steps to try to figure out where the problem is?

Sorted. Read a few more responses.

Seems a new rate limiter is in place.

How annoying.

**Comment by guillaumewantz** [comment]
Same for me. It worked for a day, then, since a few days, I can read info (fetch them in HA), but I can't send command anymore... Any idea what I need to do ?  
I am still relatively new in HA and was about to automate solar charging of my Tesla... Thanks in advance.

**Reply by Boogmeister** [reply]
> Same for me. It worked for a day, then, since a few days, I can read info (fetch them in HA), but I can't send command anymore... Any idea what I need to do ? I am still relatively new in HA and was about to automate solar charging of my Tesla... Thanks in advance.

Give up on it.

It's rate limited now so can't do what you want.

Do you not have a smart charger? I went and OCCP model so it doesn't matter what is connected.

**Reply by skipishere** [reply]
Are you using the fleet api? If not then you need to (but as mentioned, there are rate limits that apply such as only 5 charger commands a day per car)
